config/locales/de.yml

```diff
@@ -226,6 +228,8 @@ de:
       relative:
         <<: *id-generic_keys
         relation: Verwandtschaftsbeziehung
+      reminder:
+        sent_at: Gesendet am
```

Comment:
Should be: `Versandt am`
[Duden: versenden](http://www.duden.de/rechtschreibung/versenden)

Reply:
Since the action verb is send, why should we use versenden here?

Reply:
sent is past.

send = senden
sent = versandt

Reply:
Ok, then at some point I should make a PR at the future kids because they have it like this.